docs/OSD Profiles.md

```diff
@@ -0,0 +1,84 @@
+# OSD Profiles
+
+An OSD Profile is screen or page of OSD elements.  Four OSD profiles are supported, i.e. you can configure 4 different
+OSD pages each with their own OSD elements.  Elements may also be on all four pages.  With the OSD Profiles feature
```

Comment:
This will break if the documentation is reformatted to a different page width. Please change this to use only one line per paragraph in the source.

Reply:
Good point, fixed.